apps/src/applab/gridUtils.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * A couple of utility functions for dealing with our design mode grid.
+ */
+module.exports = {
```

Comment:
I'm fine with it, but out of curiosity, why this module structure instead of the flatter
```
exports.GRID_SIZE = 5;
exports.scaledDropPoint = function (draggingElement) {...
exports.snapToGridSize = function (coordinate) {...
```